docs/tools/vdb_table/data/azurecosmosdbmongodbvcore.json

```diff
@@ -0,0 +1,156 @@
+{
+    "name": "Azure Cosmos DB for MongoDB vCore",
+    "links": {
+      "docs": "https://learn.microsoft.com/azure/cosmos-db/mongodb/vcore/",
+      "github": "",
+      "website": "",
+      "vendor_discussion": "",
+      "poc_github": "",
+      "slug": ""
+    },
+    "oss": {
+      "support": "none",
+      "source_url": "",
+      "comment": ""
+    },
+    "license": {
+      "value": "Proprietary",
+      "source_url": "",
+      "comment": ""
+    },
+    "dev_languages": {
```

Comment:
This is the development language of the database itself. It was advised to me to only have 1 or 2 maximum primary development languages for this part earlier and you may want to update it as such. TY!